admin/views/school/school-home.ejs

```diff
@@ -38,13 +38,43 @@
             </li>
 
             <li>
+                <% if (pinGenerationEligibilityData.isFamiliarisationPinGenerationAllowed && !pinGenerationEligibilityData.isUnavailableTime) { %>
```

Comment:
Why not just have one flag to check?   It should just be `isFamiliarisationPinGenerationAllowed`?  Why complicate matters?